src/tls.c

```diff
@@ -10266,6 +10269,90 @@ int TLSX_EarlyData_Use(WOLFSSL* ssl, word32 maxSz)
 
 #endif
 
+/******************************************************************************/
+/* QUIC transport parameter extension                                         */
+/******************************************************************************/
+#ifdef WOLFSSL_QUIC
+
+static word16 TLSX_QuicTP_GetSize(TLSX* extension)
+{
+    const QuicTransportParam *tp = extension->data;
```

Comment:
'*' after type, not before variable name.

Reply:
This hurts, but I see now that all the code follows this. Will change.

Reply:
Did me too!
Leave quic.c alone.

Reply:
Too late, already done. ;)